src/index.html

```diff
@@ -22,6 +22,352 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="../src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
```

Comment:
```suggestion
      <nav class="nav">
```
Consider resetting the default browser styling. Also. the first link shouldn't have the left padding or margin
<img width="795" alt="Screenshot 2024-04-29 at 14 11 33" src="https://github.com/mate-academy/layout_catalog/assets/135232001/c15dc2ae-639b-4046-9c0a-2455c2ce352d">